python/daqconf/apps/trigger_gen.py

```diff
@@ -340,7 +345,22 @@ def get_trigger_app(
                                                        time_after=TRIGGER_WINDOW_AFTER_TICKS),
                                          hsi_trigger_type_passthrough=HSI_TRIGGER_TYPE_PASSTHROUGH,
                                          prescale=TTCM_PRESCALE))]
-    
+
+       if USE_FAKE_HSI_INPUT:
+        modules += [DAQModule(name = 'ttcm_fake',
+                          plugin = 'TimingTriggerCandidateMaker',
+                          conf=ttcm.Conf(s0=ttcm.map_t(signal_type=0,
+                                                       time_before=TRIGGER_WINDOW_BEFORE_TICKS,
+                                                       time_after=TRIGGER_WINDOW_AFTER_TICKS),
+                                         s1=ttcm.map_t(signal_type=TTCM_S1,
+                                                       time_before=TRIGGER_WINDOW_BEFORE_TICKS,
+                                                       time_after=TRIGGER_WINDOW_AFTER_TICKS),
+                                         s2=ttcm.map_t(signal_type=TTCM_S2,
+                                                       time_before=TRIGGER_WINDOW_BEFORE_TICKS,
+                                                       time_after=TRIGGER_WINDOW_AFTER_TICKS),
+                                         hsi_trigger_type_passthrough=HSI_TRIGGER_TYPE_PASSTHROUGH,
+                                         prescale=TTCM_PRESCALE))]
```

Comment:
Not for this PR, but I assume we might need separate config inputs for `hsi` vs `fake_hsi`... we'll deal with this once it's needed tho :/